ci/azure-pipelines.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - checkout: self
       - script: make unit-test
         displayName: Run unit test
-      - script: bash <(curl https://codecov.io/bash) -t $CODECOV_UPLOAD_TOKEN
```

Comment:
?

Reply:
no need for CODECOV_UPLOAD_TOKEN

Reply:
code cov will work with azure

Reply:
Note: Token not required for repositories uploading from Travis, CircleCI, AppVeyor, Azure Pipelines or GitHub Actions.